<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Час сортування:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="pushButtonAddUser">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Додати нового</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    *натисніть на заголовок стовпця для сортування по відповідному критерію</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>520</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ПІБ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Статус звернення</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Тип звернення</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Паспорт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата талону</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер спроби</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Решетняк Всеслава Іванівна</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>Прийнято</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>Теоретичний іспит</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>000125451</string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>380682146001</string>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string>12:00 12.12.2024</string>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сервісний центр МВС</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxSortMethod">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Пірамідальне</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Бульбашкове</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Вибором</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Метод сортування:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditSortTime">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>70</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Таблиця</string>
    </property>
    <addaction name="new_table"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Додати</string>
    </property>
    <addaction name="add_from_file"/>
    <addaction name="add_one"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="new_table">
   <property name="text">
    <string>Нова таблиця</string>
   </property>
   <property name="toolTip">
    <string>Створює нову пусту таблицю.</string>
   </property>
  </action>
  <action name="add_from_file">
   <property name="text">
    <string>Додати з файлу</string>
   </property>
   <property name="toolTip">
    <string>Додає користувачів з файлу</string>
   </property>
  </action>
  <action name="add_one">
   <property name="text">
    <string>Додати вручну</string>
   </property>
   <property name="toolTip">
    <string>Додає користувача вручну</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
